hooks/useWallet.js

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from "react";
+import { useAccount } from "wagmi";
+import { isMember } from "../utils";
+import { toast } from "react-hot-toast";
+
+const useWallet = () => {
+  const { address, isConnecting } = useAccount();
+  const [allowed, setAllowed] = useState(false);
+  const [loading, setLoading] = useState(true);
```

Comment:
Can you rename this states to `isAllowed `and `isLoading `to make it consistent with `isConnecting` from the hook `useAccount`?

Reply:
thats actually a good idea :D